layer/layer_store.go

```diff
@@ -899,6 +918,74 @@ func (ls *layerStore) deleteUnreferencedDriverLayers(ids []string) int {
 	return total
 }
 
+func (ls *layerStore) findUnusedOverlayFiles() ([]string, error) {
+	var fileNameOverlayIdRegex = regexp.MustCompile("^([a-z0-9]+).*")
+	var overlaysDir = "/var/lib/docker/overlay2"
```

Comment:
we would need to take this path from the graphdriver configuration https://github.com/balena-os/balena-engine/blob/857521b7801b4a6abf02f8da4673e868cfbe702a/layer/layer_store.go#L68

which might make this a little painful since I don't think you can get it from the `graphdriver.Driver` interface

I would probably hijack Roman's `InspectableDriver` https://github.com/balena-os/balena-engine/blob/857521b7801b4a6abf02f8da4673e868cfbe702a/daemon/graphdriver/driver.go#L108-L113 and add something like `GetRoot` there...

or another option would be to move this to the driver package altogether since it's kind of specific to it anyway